samples/sp_whoIsActive/typings/sqlops.proposed.d.ts

```diff
@@ -0,0 +1,352 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
We shouldn't have to check this file in to this extension's source code. Isn't that what the 'copytypings' step is for above?

Reply:
👍 you need to add this file to your .gitignore. To do this:
- Delete the file and check in that commit. It's already checked-in, so you need this or it'll alway have this old version.
- Add to the gitignore, re-run the copyTypings task and you shouldn't see this added into the change list